### PR TITLE
[LAM-141] Central & Service VM provisioner/manager

### PR DESCRIPTION
**Pull request by ioantsaf**
- [x] Common python code that provisions a single VM, with an external IP attached
- [x] Common code that starts, stops, destroys the VM
- [x] Manager for central service VM (provision VM, run central service ansible)
- [x] Manager for lambda service VM (provision VM, run lambda service ansible)
- [x] Add wait_for_ssh roles in service VMs ansible
- [x] Add main accepting CLI args for central/service vm provisioning and start/stop operations.

### CODE REVIEW

**Comment by ioantsaf** [comment]
please review @stevelaskaridis @gouzouni625

**Comment by stevelaskaridis** [comment]
Is this a WIP or not?

File: webapp/manager/service_vm_manager.py [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+        sm.central_service_create(vcpus=args.vcpus, ram=args.ram, disk=args.disk,
+                                  project_name=args.project_name,
+                                  public_key_path=args.public_key_path)
+    elif args.vm_id is None:
+        raise ValueError("VM id must be specified")
+    else:
+        if args.action == 'start':
+            sm.central_service_start(vm_id=args.vm_id)
+        elif args.action == 'stop':
+            sm.central_service_start(vm_id=args.vm_id)
```
Comment (stevelaskaridis) [inline webapp/manager/service_vm_manager.py]: This should be **stop** instead of **start**.

**Comment by stevelaskaridis** [comment]
When trying to destroy, with the command:
`python central_service_manager.py --action destroy --vm_id <vm_id>`

I get the following result:

```
Traceback (most recent call last):
  File "central_service_manager.py", line 115, in <module>
    csm.central_service_destroy(vm_id=args.vm_id)
  File "central_service_manager.py", line 63, in central_service_destroy
    vmmanager.destroy(vm_id=vm_id)
  File "/[...]/okeanos-LoD/core/fokia/vm_manager.py", line 128, in destroy
    if int(ip['instance_id']) == vm_id:
TypeError: int() argument must be a string or a number, not 'NoneType'
```

**Comment by stevelaskaridis** [comment]
In case there is no keys under .ssh and no path is provided, the program exits with
`IOError: [Errno 2] No such file or directory: '[...]/.ssh/id_rsa.pub'`

I think there should be a more explicit custom error.

**Comment by stevelaskaridis** [comment]
+1

**Pull request merged**